--- origsrc/gtk+-1.2.10/configure.in	2008-07-04 00:50:15.609375000 -0500
+++ src/gtk+-1.2.10/configure.in	2008-07-04 01:16:40.281250000 -0500
@@ -174,7 +174,7 @@
 AC_SUBST(STRIP_END)
 
 # i18n stuff
-ALL_LINGUAS="az ca cs da de el es et eu fi fr ga gl hr hu it ja ko lt lv nl no nn pl pt pt_BR ro ru sk sl sp sr sv tr uk vi wa zh_CN.GB2312 zh_TW.Big5"
+ALL_LINGUAS="az ca cs da de el es et eu fi fr ga gl hr hu it ja ko lt nl no nn pl pt pt_BR ro ru sk sl sp sr sv tr uk vi wa zh_CN.GB2312 zh_TW.Big5"
 AM_GTK_GNU_GETTEXT
 LIBS="$LIBS $INTLLIBS"
 
--- origsrc/gtk+-1.2.10/gdk/Makefile.am	2008-07-04 00:50:15.609375000 -0500
+++ src/gtk+-1.2.10/gdk/Makefile.am	2008-07-04 00:55:27.859375000 -0500
@@ -20,6 +20,7 @@
 libgdk_la_LDFLAGS = @STRIP_BEGIN@ \
 	-version-info $(LT_CURRENT):$(LT_REVISION):$(LT_AGE) \
 	-release $(LT_RELEASE) \
+	-no-undefined \
 	-export-dynamic \
 	@x_ldflags@	\
 @STRIP_END@
--- origsrc/gtk+-1.2.10/gtk/Makefile.am	2008-07-04 00:50:15.625000000 -0500
+++ src/gtk+-1.2.10/gtk/Makefile.am	2008-07-04 00:55:35.984375000 -0500
@@ -26,6 +26,7 @@
 libgtk_la_LDFLAGS = @STRIP_BEGIN@ \
 	-version-info $(LT_CURRENT):$(LT_REVISION):$(LT_AGE)	\
 	-release $(LT_RELEASE) 					\
+	-no-undefined \
 	-export-dynamic						\
 	@x_ldflags@						\
 @STRIP_END@
--- origsrc/gtk+-1.2.10/gtk/gtkmain.c	2001-02-27 12:29:58.000000000 -0600
+++ src/gtk+-1.2.10/gtk/gtkmain.c	2008-07-04 01:51:12.781250000 -0500
@@ -384,13 +384,22 @@
       module_name = slist->data;
       slist->data = NULL;
       if (!(module_name[0] == '/' ||
+#ifdef __CYGWIN__
+	    (module_name[0] == 'c' &&
+	     module_name[1] == 'y' &&
+	     module_name[2] == 'g') ||
+#endif
 	    (module_name[0] == 'l' &&
 	     module_name[1] == 'i' &&
 	     module_name[2] == 'b')))
 	{
 	  gchar *old = module_name;
 	  
+#ifdef __CYGWIN__
+	  module_name = g_strconcat ("cyg", module_name, ".dll", NULL);
+#else
 	  module_name = g_strconcat ("lib", module_name, ".so", NULL);
+#endif
 	  g_free (old);
 	}
       if (g_module_supported ())
--- origsrc/gtk+-1.2.10/gtk/gtkthemes.c	2000-01-21 17:22:50.000000000 -0600
+++ src/gtk+-1.2.10/gtk/gtkthemes.c	2008-07-04 01:52:23.906250000 -0500
@@ -71,7 +71,11 @@
        gchar *engine_path;
        GModule *library;
       
+#ifdef __CYGWIN__
+       g_snprintf (fullname, 1024, "cyg%s.dll", name);
+#else
        g_snprintf (fullname, 1024, "lib%s.so", name);
+#endif
        engine_path = gtk_rc_find_module_in_path (fullname);
 
        if (!engine_path)
